contrib/kube-prometheus/jsonnet/kube-prometheus/kube-prometheus.libsonnet

```diff
@@ -57,6 +57,8 @@ local configMapList = k.core.v1.configMapList;
     prometheusSelector: 'job="prometheus-k8s"',
     prometheusOperatorSelector: 'job="prometheus-operator"',
 
+    enableAdminAPI: 'false',
```

Comment:
jsonnet has merge semantics, therefore we don't need to specify this here at all